sc/shared/src/test/scala/sigmastate/lang/SigmaTemplateCompilerTest.scala

```diff
@@ -0,0 +1,55 @@
+package sigmastate.lang
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.propspec.AnyPropSpec
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import sigmastate.Values
+import sigmastate.SInt
+import sigmastate.SString
+import sigmastate.SLong
+import org.ergoplatform.sdk.Parameter
+import sigmastate.eval.CompiletimeIRContext
+import org.ergoplatform.sdk.NetworkType
+
+class SigmaTemplateCompilerTest extends AnyPropSpec with ScalaCheckPropertyChecks with Matchers {
+  property("parses docstring in example") {
+    val source =
+      """/** This is my contracts description.
+        |* Here is another line describing what it does in more detail.
+        |*
+        |* @param p1 describe p1
+        |* @param p2 description of the 2nd parameter
+        |* which is pretty complex and on many
+        |* lines to describe functions
+        |* @param param3 the final parameter
+        |* @return
+        |*/
+        |@contract def contractName(p1: Int = 5, p2: String = "default string", param3: Long) = {
+        |  sigmaProp(true)
+        |}""".stripMargin
```

Comment:
does it support 
```scala
@contract def contractName(p1: Int = 5, p2: String = "default string", param3: Long) = sigmaProp(true)
```
syntax without braces? Please add the test case.

Reply:
done